src/__experimental__/components/fieldset/fieldset-examples.stories.mdx

```diff
@@ -80,7 +80,7 @@ Address Fieldset for EU.
         label='Postal Code'
         labelInline
         labelAlign='right'
-        styleOverride={ { input: { width: '120px', flex: 'none' } } }
+        inputWidth={10}
```

Comment:
ditto

Reply:
The old input width included the label. This is the actual input width.